CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ## 0.38.0 - master
 
+* added dig command info to show if txt record is deployed
```

Comment:
I think that `TXT` should be capitalized in documentation; compare

https://en.wikipedia.org/wiki/TXT_record

(I don't think this is necessary in the command, though.)